### PR TITLE
[All hosts] (OfficeTheme) Clean up articles

### PR DESCRIPTION
**Pull request by samantharamon**
- Adds a link to the OfficeTheme APIs.
- Applies minor formatting fixes.

### CODE REVIEW

File: docs/design/add-in-color.md [hunk 1]
```diff
@@ -12,14 +12,14 @@ Color is often used to emphasize brand and reinforce visual hierarchy. It helps
...
+In Office applications, customers personalize their interfaces by applying an [Office UI theme](https://support.microsoft.com/office/365-63e65e1c-08d4-4dea-820e-335f54672310). Customers can choose between four UI themes to vary styling of backgrounds and buttons in Excel, Outlook, PowerPoint, Word, and other apps in the Office suite. To make your add-ins feel like a natural part of Office and respond to personalization, use our [Theming APIs](/javascript/api/office/office.officetheme). For example, task pane background colors switch to a dark gray in some themes. With the Theming APIs, follow suit and adjust foreground text to ensure [accessibility](../design/accessibility-guidelines.md).
```
Comment (AlexJerabek) [inline docs/design/add-in-color.md:17]: I believe this "can" is optional.

```suggestion
In Office applications, customers personalize their interfaces by applying an [Office UI theme](https://support.microsoft.com/office/365-63e65e1c-08d4-4dea-820e-335f54672310). Customers choose between four UI themes to vary styling of backgrounds and buttons in Excel, Outlook, PowerPoint, Word, and other apps in the Office suite. To make your add-ins feel like a natural part of Office and respond to personalization, use our [Theming APIs](/javascript/api/office/office.officetheme). For example, task pane background colors switch to a dark gray in some themes. With the Theming APIs, follow suit and adjust foreground text to ensure [accessibility](../design/accessibility-guidelines.md).
```

File: docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Use document themes in your PowerPoint add-ins
-description: Learn how to visually coordinate themes such as fonts and colors to apply to presentations, documents, worksheets, and emails.
+description: Learn how to visually coordinate themes, such as fonts and colors, to apply to PowerPoint presentations.
 ms.date: 07/08/2021
```
Comment (AlexJerabek) [inline docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md:4]:
```suggestion
ms.date: 05/24/2024
```

**Review by AlexJerabek — APPROVED** [review]
Approving, though I'm curious as to why there's a separate PPT article. What makes PowerPoint different from Excel or Word?

**Comment by samantharamon** [comment]
> Approving, though I'm curious as to why there's a separate PPT article. What makes PowerPoint different from Excel or Word?

I have a backlog item to expand this article to include other hosts.

**Pull request merged**